src/App.tsx

```diff
@@ -1,61 +1,135 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
```

Comment:
Consider removing commented out code. It can be confusing and add clutter to the code. 🗑️